examples/elementtree/pickle_type.py

```diff
@@ -76,4 +77,7 @@ def __init__(self, name, element):
 document = session.query(Document).filter_by(filename="test.xml").first()
 
 # print
-document.element.write(sys.stdout)
+if py3k:
```

Comment:
so, we need to make this change in optimized_ai and adjacency_list as well, right?  those still raise a "bytes" error.

Reply:
My fault, I was looking through the peephole from the issue. Addressing...

Reply:
thanks for sticking to it !   :)